src/components/LearnFooter/LearnFooter.module.css

```diff
@@ -102,3 +104,20 @@
 .NavigationStepStatusIconError {
   color: var(--error);
 }
+
+.FooterCopyright {
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  margin-top: 10px;
+}
+
+.FooterCopyrightLink {
+  text-decoration: none;
+  user-select: none;
+  color: var(--txt-2);
+}
+
+.FooterCopyrightLink:hover {
+  opacity: 0.8;
+}
```

Comment:
Your pull request must not contain any changes to this file.

Reply:
Solved.